dev/devicelab/bin/tasks/ios_app_with_extensions_test.dart

```diff
@@ -39,11 +39,9 @@ Future<void> main() async {
           'build',
           options: <String>['ios', '--no-codesign', '--release', '--verbose'],
         );
-        if (!buildOutput.contains('Watch companion app found')) {
-          throw TaskResult.failure('Did not detect watch companion');
-        }
-        if (buildOutput.contains('-sdk iphoneos -destination')) {
-          throw TaskResult.failure('-sdk must be omitted for app with watch companion');
+        if (!buildOutput.contains('-destination generic/platform=watchOS')) {
+          print(buildOutput);
+          throw TaskResult.failure('Did not try to get watch build settings');
```

Comment:
I think you also need to delete the rest of this test, because the build won't actually succeed the `Runner.app` files won't exist.

Reply:
Fixed in https://github.com/flutter/flutter/pull/114078/commits/df3951741ef47c486a7be950099153004e857c5c.